features/ui/spinner/spinner.tsx

```diff
@@ -3,24 +3,26 @@ import { color, space } from "@styles/theme";
 
 const Container = styled.div`
   width: 100%;
-  height: 25vh;
+  height: 27vh;
   display: flex;
   justify-content: center;
   align-items: center;
 `;
 
 const spin = keyframes`
   to {
+    transform: rotate(360deg);
```

Comment:
Fyi you don't need to add the `-webkit-transform` and `-webkit-animation` below. styled-components comes with an auto-prefixer and adds these things automatically.

Reply:
Good to know!